koku/masu/external/kafka_msg_handler.py

```diff
@@ -546,29 +549,39 @@ def process_report(report):
     complete.
 
     Args:
+        request_id (Str): The request id
         report (Dict) - keys: value
+                        request_id: String,
+                        account: String,
                         schema_name: String,
                         manifest_id: Integer,
                         provider_uuid: String,
                         provider_type: String,
                         current_file: String,
+                        date: DateTime
 
     Returns:
         True if line item report processing is complete.
 
     """
+    account = report.get("account")
     schema_name = report.get("schema_name")
     manifest_id = report.get("manifest_id")
     provider_uuid = report.get("provider_uuid")
     provider_type = report.get("provider_type")
+    start_date = report.get("date")
 
     report_dict = {
         "file": report.get("current_file"),
         "compression": UNCOMPRESSED,
         "manifest_id": manifest_id,
         "provider_uuid": provider_uuid,
     }
-    return _process_report_file(schema_name, provider_type, provider_uuid, report_dict)
+    result = _process_report_file(schema_name, provider_type, provider_uuid, report_dict)
```

Comment:
I think we're going to have to pull the `copy_local_report_file_to_s3_bucket` logic out of the downloader or use the OCP downloader here and call it.  Re-using the downloader logic would help to make it so OCP local could still work...

The recent multi-file OCP payload logic change made it so the OCP downloader isn't actually calling `download_file` anymore.  It's only used for OCP-local.  Sorry for the confusion on that!

Reply:
Another option might be to consolidate and move the `copy_local_report_file_to_s3_bucket` logic from the downloaders and put it into something that could be called in or after `_get_report_files`.  Then that same logic could also be called from the listener.

Reply:
I'm extracting out a function `create_daily_archives` that can be called in the `download_file` and the `extract_payload` flows.

Reply:
Update pushed.